benchmarks/iter.cpp

```diff
@@ -19,7 +19,7 @@ int main() {
     }
     timer.stop();
     double flops = 10.0*512*512*512*2;
-    double seconds = timer.elapsedSeconds();
+    double seconds = timer.elapsed();
```

Comment:
What's the reason for this?  (I'm assuming that elapsed() is more standard and more likely to work on modern compilers?)

Reply:
Well my compiler g++ 8.2.1 complained. I didi it mechanically to allow the test to compile. I must say I was even thinking it was a change internal to blitz (I though it had its own timer class)...

Reply:
I see.... looks like it was just a bug, and nobody had bothered to compile `benchmarks/iter.cpp`, which depends on Blitz's `timer.h`.

Could we ditch `timer.h` and use `std::chrono` from C++11?  In general, anything that we can get rid of in favor of now-standard C++ features is good (as long as it doesn't change the API; that is for later).

https://en.cppreference.com/w/cpp/chrono

As I said before... I don't care to maintain pre-C++11 compatibility going forward.  We can go ahead and change the code along with the build system, as long as it doesn't change the API.

Reply:
+ 1
See my comment from this morning, I even believe that we can suppress the library (or at least most of its required uses for non debugging purposes) by using constexpr... Not sure whether suppressing the need for the library counts as an API change (people can always link with a non-necessary library)...